docs/spec/blockchain/blockchain.md

```diff
@@ -56,8 +56,8 @@ type Header struct {
     ConsensusParamsHash []byte  // SimpleMerkle of the ConsensusParams
 
     // consensus
-    Proposer            []byte  // Address of the block proposer
```

Comment:
I think `ProposerAddress` better reflects the field's value. It's really an address (without pubkey or power).